addons/gut/plugin_control.gd

```diff
@@ -73,8 +73,10 @@ export var _disable_strict_datatype_checks = false
 export var _test_prefix = 'test_'
 # The prefix used to find test scripts.
 export var _file_prefix = 'test_'
+# The suffix used to find test scripts.
+export var _file_suffix = '.gd'
 # The file extension for test scripts (I don't think you can change this and
-# everythign work).
+# everything will work).
 export var _file_extension = '.gd'
```

Comment:
I wasn't sure if this option would still be necessary. You include it as an option for configuring GUT, however this comment make me think it actually can't be changed.

With the new `_file_suffix` variable added above, this `_file_extension` variable is actually unnecessary. It is still passed to `gut.gd` but it is no longer used by it anywhere.

Do you have a preference for how to use this variable? I'd recommend removing it entirely since `_file_suffix` accomplishes the same goal, but `suffix` is a better name for it.

I can make the change in this PR if you'd like.